### ISSUE TITLE
Add bowling scoring plugin

### ISSUE THREAD

**Issue by jonaskoelker** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#522]
Market research suggests the rising Chinese market has an increasing interest in bowling.

Please implement a bowling scoring plugin which we can upsell as an add-on.

Thank you.

**Comment by JulianGmp** [comment]
I think it would be important to allow the add-on to be interfaced using a HTTP REST interface

**Comment by patrislav1** [comment]
We can unlock this feature with a in-app purchasing system, and save the bowling high-scores in a blockchain.

**Comment by arunanshub** [comment]
@patrislav1 
> We can unlock this feature with a in-app purchasing system, and save the bowling high-scores in a blockchain.

I recommend looking into EA's work in this field. They have ***[Unlock the rest of the comment for $420.69]***

**Comment by jonaskoelker** [comment]
This appears to overlap in functionality with #475.

Let's hold a meeting to decide whether to mark this as "duplicate of #475" or "depends on #475".
